lib/vmfloaty.rb

```diff
@@ -214,11 +214,11 @@ def run # rubocop:disable Metrics/AbcSize
         if delete_all
           running_vms = service.list_active(verbose)
           if running_vms.empty?
-            STDERR.puts 'You have no running VMs.'
+            STDOUT.puts 'You have no running VMs.'
```

Comment:
This should be an error. Trying to delete everything with no running vms might be something other tools have relied on, and changing what output gets printing might end up breaking those users tools.